doc/guides/SimpleStartupGuide.md

```diff
@@ -434,67 +463,58 @@ There are multiple commands that can update aspects of the camera entry in meta-
    profileToken: profile_2
    ```
 
-2. Convert the JSON input to Base64:
+2. Execute the `GetStreamURI` command to get RTSP URI from the ONVIF device.  
 
-   >NOTE: Make sure to change the profile token to the one found in step 1. In this example, it is the string `profile_1`.
-
-   ```json
-   {
-      "ProfileToken": "profile_1"
-   }
-   ```
-   Example Output:
+   >NOTE: Make sure to change the profile token to the one found in step 1. In this example, it is the string `profile_1`.  
 
    ```bash
-   echo -n '{
-      "ProfileToken": "profile_1"
-   }' | base64
-   ewogICAgICAiUHJvZmlsZVRva2VuIjogInByb2ZpbGVfMSIKfQ==
-   ```
-
-3. Execute the `GetStreamURI` command to get RTSP URI from the ONVIF device. Make sure to put the Base64 JSON data after *?jsonObject=* in the command.
-
-   ```bash
-   curl -s http://0.0.0.0:59882/api/v2/device/name/Camera001/StreamUri?jsonObject=ewogICAgICAiUHJvZmlsZVRva2VuIjogInByb2ZpbGVfMSIKfQ== | jq -r '"streamURI: " + '.event.readings[].objectValue.MediaUri.Uri''
+      curl -s "http://0.0.0.0:59882/api/v2/device/name/Camera001/StreamUri?jsonObject=$(base64 -w 0 <<< '{
+         "StreamSetup" : {
+            "Stream" : "RTP-Unicast",
+            "Transport" : {
+               "Protocol" : "RTSP"
+            }
+         },
+         "ProfileToken": "profile_1"
+      }')" | jq -r '"streamURI: " + '.event.readings[].objectValue.MediaUri.Uri''
    ```
    
    Example Output:
 
    ```bash
-      % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
-                                 Dload  Upload   Total   Spent    Left  Speed
-   100   553  100   553    0     0  21269      0 --:--:-- --:--:-- --:--:-- 21269
    streamURI: rtsp://192.168.86.34:554/stream1
    ``` 
 
-4. Stream the RTSP stream. 
```

Comment:
@ChristianDarr-personal Here's another example where we have a dot instead of a colon
https://github.com/edgexfoundry/device-onvif-camera/pull/161#discussion_r973302960